instrumentation/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/r2dbc/R2DbcInstrumentationAutoConfigurationTest.java

```diff
@@ -31,7 +36,43 @@ class R2DbcInstrumentationAutoConfigurationTest {
           .withConfiguration(
               AutoConfigurations.of(
                   R2dbcInstrumentationAutoConfiguration.class, R2dbcAutoConfiguration.class))
-          .withBean("openTelemetry", OpenTelemetry.class, testing::getOpenTelemetry);
+          .withBean(
+              "openTelemetry",
+              OpenTelemetry.class,
+              R2DbcInstrumentationAutoConfigurationTest::openTelemetry);
+
+  private static OpenTelemetry openTelemetry() {
```

Comment:
an alternative would be to move this wrapping to the `LibraryTestRunner` @trask WDYT?

Reply:
moving it makes sense to me, since `LibraryTestRunner` manages the SDK lifecycle